modules/psap-special-resource-operator-installing.adoc

```diff
@@ -0,0 +1,171 @@
+// Module included in the following assemblies:
+//
+// * hardware_enablement/psap-special-resource-operator.adoc
+
+[id="installing-the-special-resource-operator_{context}"]
+= Installing the Special Resource Operator
+
+As a cluster administrator, you can install the Special Resource Operator (SRO) using the {product-title} CLI or the web console.
+
+[id="install-operator-cli_{context}"]
+== Installing the Special Resource Operator using the CLI
+
+As a cluster administrator, you can install SRO using the CLI.
+
+.Prerequisites
+
+* You have a running {product-title} cluster
+* You have installed the OpenShift CLI (`oc`).
+* You are logged into the OpenShift CLI as a user with `cluster-admin` privileges.
+* You have installed the xref:../psap-node-feature-discovery#installing-the-node-feature-discovery-operator_node-feature-discovery-operator[Node Feature Discovery (NFD) Operator]
+
+.Procedure
+
+. Create a namespace for the Special Resource Operator.
+
+.. Create the following `Namespace` custom resource (CR) that defines the `openshift-special-resource-operator` namespace, and then save the YAML in the `sro-namespace.yaml` file:
++
+[source,yaml]
+----
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: openshift-special-resource-operator
+----
+
+.. Create the namespace by running the following command:
++
+[source,terminal]
+----
+$ oc create -f sro-namespace.yaml
+----
+
+. Install SRO in the namespace you created in the previous step.
+
+.. Create the `OperatorGroup` CR and save the YAML in the `sro-operatorgroup.yaml` file:
++
+[source,yaml]
+----
+apiVersion: operators.coreos.com/v1
+kind: OperatorGroup
+metadata:
+  generateName: openshift-special-resource-operator-
+  name: openshift-special-resource-operator
+  namespace: openshift-special-resource-operator
+spec:
+  targetNamespaces:
+  - openshift-special-resource-operator
+----
+
+.. Create the `OperatorGroup` CR by running the following command:
```

Comment:
Run the following `oc create` command to create the `OperatorGroup` CR:

(note - there are backticks around command)